### ISSUE TITLE
Default keymaps are not removed (Windows)

### ISSUE THREAD

**Issue by sho-87** [kiyoon/jupynium.nvim#41]
I have turned default keymaps off in my config and have setup a few new binds:

```
use_default_keybindings = false,
textobjects = {
    use_default_keybindings = false,
},
```
I can see my new custom binds OK, but the default maps still show up in my which-key and seems like they are still being set? Presumably this has something to do with me not having any replacement binds for the default `<space>x`, for example

![image](https://user-images.githubusercontent.com/5199715/220513395-f30dac6b-c0b9-4fdc-8ee1-941e0d096144.png)

My config:

- Binds: https://github.com/sho-87/dotfiles/blob/master/nvim/lua/keybinds.lua#L204
- Jupynium: https://github.com/sho-87/dotfiles/blob/master/nvim/lua/plugins/modules/jupynium.lua

**Comment by kiyoon** [comment]
Weird, I just tested on Windows and to me it works fine not mapping those.

Does any of the customisation seem to work? For example, if you set shortsighted do you see cells being dimmed? I don't know if it's a problem of keymaps and auto start, or it's just the entire config being ignored.

**Comment by sho-87** [comment]
fixed in #42
